mirascope/core/base/stream.py

```diff
@@ -148,7 +148,7 @@ def __aiter__(
         """Iterates over the stream and stores useful information."""
         self.content = ""
 
-        async def generator():
+        async def generator() -> AsyncGenerator[tuple[Any, Any], Any]:
```

Comment:
`Any` is too broad here, should be more specific, likely using generic types? or not worth?